backend/kafka/create_topic.py

```
@@ -1,38 +1,177 @@
+        return False
+
+    try:
+        existing_topics = admin_client.list_topics()
+
+        print(f"[INFO] Existing topics: {existing_topics}")
+
+    except KafkaError as e:
+        print(f"[ERROR] Failed to fetch topic list: {e}")
+        try:
+            admin_client.close()
+        except Exception as close_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)